server/server.ts

```diff
@@ -40,29 +40,28 @@ if (conf.DOMAIN === 'thegulocal.com') {
 
 server.use(helmet());
 
-if (featureSwitches.cspSecurityAudit) {
-	const cspDefaultSrcAllowList = [
-		"'self'",
-		'https://sourcepoint.theguardian.com',
-		'https://gnm-app.quantummetric.com',
-		'https://assets.guim.co.uk',
-		'https://ophan.theguardian.com',
-	].join(' ');
-	const csp = [
-		'report-uri /api/csp-audit-report-endpoint',
-		'report-to csp-endpoint',
-		`default-src ${cspDefaultSrcAllowList}`,
-		`style-src 'unsafe-inline'`, // this is unsafe but needed for now for emotion
-	];
-	server.use(function (_: Request, res: Response, next: NextFunction) {
-		res.set({
-			'Report-To':
-				'{ "group": "csp-endpoint", "endpoints": [ { "url": "/api/csp-audit-report-endpoint" } ] }',
-			'Content-Security-Policy-Report-Only': `${csp.join('; ')};`,
-		});
-		next();
+export const createCsp = (hashes: string[]) => {
+	return `script-src ${hashes
+		.map((hash) => `'sha256-${hash}'`)
+		.join(
+			' ',
+		)} 'strict-dynamic'; style-src 'unsafe-inline'; object-src 'none'; base-uri 'none';`;
```

Comment:
I preferred the old way of building it up and then joining it
```suggestion
	const prefixedHashes = hashes.map((hash) => `'sha256-${hash}'`);
	const csp = [
		`script-src ${prefixedHashes.join(' ')} 'strict-dynamic'`,
		`style-src 'unsafe-inline'`,
		`object-src 'none'`,
	];
	return csp.join('; ');
```

Reply:
Yep, that's nicer :)